Princeton-CDH/cdh-ansible
# Update test-prosody and prosody for jammy jellyfish and NFS for media/data

changes so far:
- add cdh-test-prosody2 to prosody_qa hosts definition
- remove python-virtualenv from default ubuntu packages list (not installable); added python3 packages, and switched the one python call outside of a python virtualenv to use `python3` instead of `python`
- make stop apache2 handler conditional on the service actually being installed

remaining problems:
- I had to reload nginx manually to get the new application to respond; this is not a new problem but thought I would mention (I made a change a while back to try to set it so the handler always runs but it seems like I must have missed something)
- it appears that the new default shell for `pulsys` is `zsh` instead of `bash`; this is ok, but I would like some help updating my existing bash profile settings and alias to work with zsh. (Is it fine for conan to still be a bash user?)


question: is there a better way to manage hosts so we can run only on the new host? or is it best to run playbook on both?
